src/main/java/com/playmonumenta/networkrelay/WhereIsCommand.java

```diff
@@ -0,0 +1,36 @@
+package com.playmonumenta.networkrelay;
+
+import dev.jorel.commandapi.CommandAPI;
+import dev.jorel.commandapi.CommandAPICommand;
+import dev.jorel.commandapi.CommandPermission;
+import dev.jorel.commandapi.arguments.EntitySelectorArgument;
+import org.bukkit.entity.Player;
+
+public class WhereIsCommand {
+	public static void register() {
+		new CommandAPICommand("whereis")
+			.withPermission(CommandPermission.fromString("monumenta.networkrelay.whereis"))
+			.withArguments(new EntitySelectorArgument.OnePlayer("player"))
+			.executes((sender, args) -> {
+				Player player = (Player) args[0];
+				// TODO: REMOVE THIS COMMAND, THIS IS FOR DEBUGGING DEV/MOD only - usb
```

Comment:
This is a command requested by the moderator team. Also note that using an entity selector argument requires that you already know where the player is and run the command from their shard, which is counterproductive. This needs to use a TextArgument with suggestions for the names of visible player names - or also include hidden player names if the CommandSender has the permission to see them, but I never got around to figuring that out. An operator check is not sufficient to verify this due to the build shard.

Reply:
TODO comment was copied from a debugging command